testdata/dnn/download_models.py

```diff
@@ -1001,14 +1001,14 @@ def get_confirm_token(response):  # in case of large files
         sha='fcd0fa401c83bf2b29e18239a9c2c989c9b8669d',
         filename='onnx/models/yolov7_not_simplified.onnx'),
     Model(
-        name='NanoTrack (ONNX)',
-        url='https://raw.githubusercontent.com/zihaomu/opencv_extra_data_backup/main/NanoTrack/models/nanotrack_backbone_sim.onnx',
-        sha='9b083a2dbe10dcfe17e694879aa6749302a5888f',
+        name='NanoTrackV2 (ONNX)',
+        url='https://raw.githubusercontent.com/zihaomu/opencv_extra_data_backup/main/NanoTrackV2/models/nanotrack_backbone_sim_v2.onnx',
+        sha='6e773a364457b78574f9f63a23b0659ee8646f8f',
         filename='onnx/models/nanotrack_backbone_sim.onnx'),
     Model(
-        name='NanoTrack (ONNX)',
-        url='https://raw.githubusercontent.com/zihaomu/opencv_extra_data_backup/main/NanoTrack/models/nanotrack_head_sim.onnx',
-        sha='8fa668893b27b726f9cab6695846b4690650a199',
+        name='NanoTrackV2 (ONNX)',
```

Comment:
File names must be changed.
Old versions of files must be kept.

Reply:
Thx for your suggestion. I will keep the model and regression test of V1 and V2 at the same time.